src/sdi-refresh-monitor.c

```diff
@@ -535,12 +532,10 @@ static void manage_refresh_inhibit(SnapdClient *source, GAsyncResult *res,
   }
 }
 
-static void notice_cb(GObject *object, SnapdNotice *notice, gboolean first_run,
-                      gpointer monitor) {
-  SdiRefreshMonitor *self = SDI_REFRESH_MONITOR(monitor);
-
-  g_autoptr(GHashTable) data = snapd_notice_get_last_data(notice);
-  g_autofree gchar *kind = g_strdup(g_hash_table_lookup(data, "kind"));
+void sdi_refresh_monitor_notice(SdiRefreshMonitor *self, SnapdNotice *notice,
+                                gboolean first_run, gpointer data) {
+  g_autoptr(GHashTable) notice_data = snapd_notice_get_last_data(notice);
```

Comment:
Should be `snapd_notice_get_last_data2` now?

Reply:
Yes, but since that was changed in a different patch, I kept it this way. Now that the PR has been merged, it has already been changed.